### PR TITLE
Docs/952 visuals for overview page

### PR DESCRIPTION
**Pull request by arecuenco-dsgn**
All the visual svgs are done except accordion and breadcrumbs because the UI is still in progress.

Here is the SVG of the visuals:
[sl-overvie-visuals.json](https://github.com/sl-design-system/components/files/14297899/sl-overvie-visuals.json)

We are creating new components pages in different branches, here is the list of the SVGs that are done (include) in this branch:
- avatar
- button
- checkbox
- dialog
- popover
- radio-group
- select
- skeleton
- switch
- tab-group
- text-field
- textarea
- tooltip

This are the SVGs that are left:
- Inline Message
- Accordion
- Breadcrumbs
- Spinner
- Badge
- Card

### CODE REVIEW

**Comment by DanielleRameau** [comment]
@arecuenco-dsgn Nice! Overall I like all the visuals. I've 1 comment: I think the black bars in the skeleton state are a bit strange, can we keep it light grey there?

Screenshot 2024-02-15 at 16 29 50

File: website/src/categories/components/tooltip/tooltip.md [hunk 1]
```diff
@@ -3,13 +3,15 @@ title: Tooltip
...
...121h.106c.04-.01.096-.02.167-.03l.166.94c-.081.04-.177.071-.288.091a2.46 2.46 0 0 1-.424.03ZM158.24 106.024c-.788 0-1.339-.227-1.652-.682-.304-.455-.455-1.046-.455-1.773v-4.078h-1.092v-.94l1.152-.076.152-2.061h1.046v2.061h1.986v1.016h-1.986v4.093c0 .454.081.808.242 1.061.172.242.47.364.895.364.131 0 .273-.021.424-.061l.409-.137.243.94a5.13 5.13 0 0 1-.667.182 2.734 2.734 0 0 1-.697.091ZM161.042 105.842v-7.367h1.243v7.367h-1.243Zm.637-8.882a.885.885 0 0 1-.606-.212c-.152-.152-.228-.354-.228-.607 0-.242.076-.44.228-.591a.85.85 0 0 1 .606-.227.8.8 0 0 1 .591.227.773.773 0 0 1 .243.591c0 .253-.081.455-.243.606a.833.833 0 0 1-.591.213ZM164.773 108.95V98.475h1.03l.106.85h.046a5.69 5.69 0 0 1 1.091-.728 2.78 2.78 0 0 1 1.258-.303c.95 0 1.673.343 2.168 1.03.495.677.743 1.587.743 2.729 0 .828-.152 1.541-.455 2.137-.293.596-.682 1.051-1.167 1.364-.475.313-.996.47-1.562.47-.343 0-.687-.076-1.03-.227a5.296 5.296 0 0 1-1.016-.622l.03 1.289v2.486h-1.242Zm3.046-3.972c.607 0 1.107-.257 1.501-.773.404-.525.606-1.243.606-2.152 0-.809-.151-1.46-.455-1.956-.293-.505-.788-.757-1.485-.757-.313 0-.632.085-.955.257-.313.172-.652.42-1.016.743v3.865c.334.283.657.485.971.606.313.112.591.167.833.167Z"/></svg>
```
Comment (anna-lach) [inline website/src/categories/components/tooltip/tooltip.md:6]: It looks like for tabs there is wrong SVG added (accordion)?

image

File: website/src/categories/components/message-dialog/message-dialog.md [hunk 2]
```diff
@@ -10,6 +10,6 @@ tags: component
 eleventyNavigation:
   parent: Components
   key: Message dialog
-  order: 13
+  order: 16
```
Comment (anna-lach) [inline website/src/categories/components/message-dialog/message-dialog.md:13]: Can you please fix the order to have all components alphabetically? :)

image

File: website/src/categories/components/icon/icon.md [hunk 3]
```diff
@@ -11,6 +11,6 @@ packageName: icon
 eleventyNavigation:
   parent: Components
   key: Icon
-  order: 11
+  order: 14
   status: coming
```
Comment (anna-lach) [inline website/src/categories/components/icon/icon.md:15]: Images for form and form field are missing? :)

image

**Review by anna-lach — APPROVED** [review]

**Pull request merged**